app/src/main/res/values/arrays.xml

```diff
@@ -42,6 +42,7 @@
         <item>English (day of year only)</item>
         <item>Español</item>
         <item>Italiano</item>
+        <item>Euskara</item>
```

Comment:
Many thanks for your submission.

This particular file shouldn't be changed though. This setting for the languages is for the calendar data, which is not managed in this repository.

If you want the calendar data to be translated in basque, the translations need to be added to this repository: https://github.com/caarmen/french-revolutionary-calendar

If you only want the UI of the app to be translated into basque, the changes to this file here should be reverted.